selenium-system-test/src/test/scala/hts/pages/AuthorityWizardPage.scala

```diff
@@ -95,4 +118,26 @@ object AuthorityWizardPage extends WebPage {
   def setCountryCode(countryCode: String)(implicit driver: WebDriver): Unit =
     find(name("itmp.address.countryCode")).foreach(_.underlying.sendKeys(countryCode))
 
+  private def setAddressLines(address: Address)(implicit driver: WebDriver): Unit = {
```

Comment:
can we make this logic simple ?

Reply:
we had something like 
```scala
Try(address.lines(0)).toOption.foreach(setAddressLine1)
Try(address.lines(1)).toOption.foreach(setAddressLine2)
Try(address.lines(2)).toOption.foreach(setAddressLine3)
...
```
which uses the unsafe accessor methods on the `List` which were generating compile-time warnings. The code here was a way to avoid those warninings